README.md

```diff
@@ -402,22 +475,64 @@ Example `process_exec` and `process_exit` events can be:
 </p>
 </details>
 
-For the rest of the use cases we will use the Tetragon CLI to give the output.
+#### Use case 2: Privileged Execution
+
+Tetragon also provides the ability to check process capabilities and kernel namespaces.
+
+As a first step let's enable visibility to capability and namespace changes via the configmap
+by setting `enable-process-cred` and `enable-process-ns`  from `false` to `true`
+```bash
+kubectl edit cm -n kube-system tetragon-config
+# change "enable-process-cred" from "false" to "true"
+# change "enable-process-ns" from "false" to "true"
+# then save and exit
+```
+Restart the Tetragon daemonset:
+```
+kubectl rollout restart -n kube-system ds/tetragon
+```
+
+As a second step, let's start monitoring the Security Observability events from the privileged `test-pod` workload:
+```bash
+kubectl logs -n kube-system -l app.kubernetes.io/name=tetragon -c export-stdout -f | tetra getevents --namespace default --pod test-pod
+```
+
+In another terminal let's apply the privileged PodSpec:
+```bash
+kubectl apply -f https://raw.githubusercontent.com/cilium/tetragon/main/testdata/specs/testpod.yaml
+```
+
+If you observe the output in the first terminal, you can see the container start with `CAP_SYS_ADMIN`:
+```bash
+🚀 process default/test-pod /bin/sleep 365d                🛑 CAP_SYS_ADMIN
+🚀 process default/test-pod /usr/bin/jq -r .bundle         🛑 CAP_SYS_ADMIN
+🚀 process default/test-pod /usr/bin/cp /kind/product_name /kind/product_uuid /run/containerd/io.containerd.runtime.v2.task/k8s.io/7c7e513cd4d506417bc9d97dd9af670d94d9e84161c8c8 fdc9fa3a678289a59/rootfs/ 🛑 CAP_SYS_ADMIN
+```
+
```

Comment:
Can we do another use case "Use Case 3: Namespace access"? An easy way would be to monitor whenever the sys_setns syscall is invoked and just kubectl exec into a K8s pod, and run nsenter into different kernel namespaces. E.g:  "kubectl exec -it test-pod -- nsenter -t 1 -m -u -n -i -p bash" (in case of test-pod). And see what happens.

CRD:
```
apiVersion: cilium.io/v1alpha1
kind: TracingPolicy
metadata:
  name: "namespace-change"
spec:
  kprobes:
  - call: "__x64_sys_setns"
    syscall: true
    args:
    - index: 0
      type: "int"
```

Reply:
See https://github.com/cilium/tetragon/issues/706